packages/cli/lib/project-generator.js

```diff
@@ -217,13 +217,35 @@ module.exports = class ProjectGenerator extends BaseGenerator {
     });
   }
 
+  promptYarnInstall() {
+    if (this.shouldExit()) return false;
+    const prompts = [
+      {
+        type: 'confirm',
+        name: 'yarn',
+        message: g.f('Yarn is available. Do you prefer to use it by default?'),
+        when:
+          !this.options.packageManager &&
+          this.spawnCommandSync('which', ['yarn'], {stdio: false}).status === 0,
```

Comment:
Is `which` available on Windows? If not, maybe the following is better:

```js
this.spawnCommandSync('yarn', ['help'], {stdio: false}).status === 0,
```

Reply:
well seen